ui/src/components/search/SearchResults.tsx

```diff
@@ -15,13 +19,47 @@ const SearchResults: React.FC<SearchResultsProps> = ({
   count,
   params,
 }) => {
+  const router = useRouter();
+
+  const sortOptions = [
+    {
+      label: "Most recent",
+      value: "_created_at",
+    },
+    {
+      label: "Least recent",
+      value: "-_created_at",
```

Comment:
let's add `updated_at`, I think it's more relevant, otherwise it looks good :) thank you

Reply:
@karolina-siemieniuk-morawska ☝️